### DISCUSSION TITLE
How to change the search box width of the taskbar on win10-x64?

### DISCUSSION THREAD

**Discussion by qwertyue** [ramensoftware/windhawk#219] [General]
I noticed a SearchBoxWidth under the registry path : HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Search.

It has two values: 344 and 288, corresponding to the large taskbar and small taskbar.

But every time I change this value and restart exploerer.exe, the value will be restored to 344 or 288.

So is there any other way to change the search box width in the taskbar?

**Comment by thisiselias** [comment]
I too was bashing my head in the wall about this so I did a little bit of digging with Ghidra. Turns out that you can change the width of it by instead changing HKCU\Software\Microsoft\Windows\CurrentVersion\Search\Flighting\Override and then creating a 32-Bit value called SearchBoxWideWidth. Change to decimal and then pick your desired pixel width.

Hope this helps :)

**Reply by thisiselias** [reply]
As far as I'm aware this used to be something that actively rolled out to Windows 10 up until late last year. 344 is the default wide with and with 288 being the shorter variant (I might be wrong). If someone still has the shorter variant from back when it was actively rolling out, then please do look up the value of said registry key above and post it here.
